README.md

```diff
@@ -111,6 +111,15 @@ class MyController : public MjbotsControlLoop<LcmLog, LcmInput, MyRobot>
 Refer to `include/examples/simple_robot.h` for a sample robot class and 
 `examples/robot_example.cpp` for a usage example.
 
+## Behaviors
+The `Behavior` abstract class can be derived by the user and used to define 
+custom behaviors.  The abstract class includes functions for behavior 
+initialization, startup, updating, and stopping, as well as methods for 
```

Comment:
Provide guidelines on how to use. What do you need to do when you inherit from the behavior base?

Reply:
Added in f5235b4a9f2f70179410d9f36c16d79c5d13f04e